scripts/tokenomics-emulation/config.json

```diff
@@ -111,6 +111,9 @@
         "name": "cere-user-to-app",
         "amount": 1,
         "token_value": 1
+      },
+      {
+        "name": "total-issuance"
```

Comment:
What are you going to do with `total-issuance`? Update, Increase? Get?

Reply:
Renamed as `fetch-total-issuance`